src/App.js

```diff
@@ -1,25 +1,13 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
+import React from 'react';
+import Calc from './component/calc';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Setup Math Magicians Project using Create React App.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+/* eslint-disable react/prefer-stateless-function */
+class App extends React.Component {
```

Comment:
- As suggested earlier, kindly refactor your component to a functional component.
- All commented lines of code should be removed as well. 🤗

Reply:
-I'll definitely fix is thank you for the constructive review.